Source/WebKit/UIProcess/RemoteLayerTree/RemoteLayerTreeNode.mm

```diff
@@ -67,16 +67,29 @@
     return makeUnique<RemoteLayerTreeNode>(layerID, WTFMove(layer));
 }
 
+#if ENABLE(INTERACTION_REGIONS_IN_EVENT_REGION)
+void RemoteLayerTreeNode::applyScrollUpdatesToInteractionRegionsLayer()
+{
+    CALayer *contentLayer = layer();
+    CALayer *interactionLayer = interactionRegionsLayer();
```

Comment:
Do we always have one? Should we bail here?

Reply:
probably safer to check
(looked into lazyly creating the interaction container too, but that's for another branch :))